randomness-agent/examples/randomness/main.rs

```diff
@@ -161,13 +161,13 @@ where
 
     // Create a threshold signer
     let dst = format!(
-        "dcipher-randomness-v01-BN254G1_XMD:KECCAK-256_SVDW_RO_0x{:064x}_",
+        "dcipher-randomness-v01-BLS12381G1_XMD:SHA-256_SSWU_RO_0x{:064x}_",
```

Comment:
Are you sure you're using SHA-256 and not KECCAK-256 in the Solidity code for hash to curve and in the EIP stuff?
Usually Ethereum is using KECCAK-256 everywhere, no?

Reply:
100% sure. Ethereum has had a precompile for sha2 since the beginning, it's just 2x+ as expensive as the keccak opcode. That's dwarfed by EC costs in our case anyway. As I said in Slack, I've consistently used sha2 along with BLS12-381 to maintain drand compatibility and avoid the complexity of a 3rd signature scheme/suite.